Automattic/wp-calypso
# Untangling: Add Advanced Tools -> Staging Site

Fixes:

- https://github.com/Automattic/dotcom-forge/issues/9615

## Proposed Changes

This PR adds the Advanced Tools -> Staging site subtab under the `untangling/hosting-menu` flag.

### Production site

<img width="966" alt="image" src="https://github.com/user-attachments/assets/e939a8a9-f7c2-4eeb-9985-4bbafa89a9c7">

### Staging site

<img width="964" alt="image" src="https://github.com/user-attachments/assets/d868abc7-92ad-4f49-bef3-8babeccf0c67">

## Why are these changes being made?

pbxlJb-6ye-p2

## Testing Instructions

1. Go to /sites
2. Click an Atomic site
3. Go to Advanced Tools -> Staging site subtab.
4. Verify that this tool is working.
5. Repeat with a staging site.
6. Smoke-test staging site features; verify that the features are working.
7. Regression testing:
    - Go to /sites?flags=-untangling/hosting-menu
    - Click an Atomic site
    - Verify that the original Staging site tab is still working.
    - Repeat with a staging site.
    - Smoke-test staging site features; verify that the features are working.

## Pre-merge Checklist
